docs/content/en/docs/reference/cloudstack/cloudstack-prereq.md

```diff
@@ -5,3 +5,63 @@ weight: 15
 description: >
   CloudStack provider requirements for EKS Anywhere
 ---
+
+To run EKS Anywhere, you will need:
+
+### Prepare Administrative machine
+Set up an Administrative machine as described in [Install EKS Anywhere ]({{< relref "../../getting-started/install/" >}}).
+
+### Prepare a CloudStack environment
+
+To prepare a CloudStack environment to run EKS Anywhere, you need the following:
+
+* A CloudStack 4.14 or later environment. CloudStack 4.16 is used for examples in these docs.
+* Capacity to deploy 6-10 VMs.
+* One network in CloudStack to use for the cluster. EKS Anywhere clusters need access to CloudStack through the network to enable self-managing and storage capabilities.
+* A Red Hat Enterprise Linux ISO image built into a qcow2 image as described in [artifacts]({{< relref "../artifacts/" >}}).
+* User credentials to create VMs and attach networks in CloudStack.
```

Comment:
A cloudstack user with access and secret key ?

Reply:
We should also need to document the list of Cloudstack permissions required for the user.

Reply:
> A cloudstack user with access and secret key ?

The user needs to have an API Key and a Secret Key.

Reply:
Okay. I noted the CloudStack API key and Secret key.